### ISSUE TITLE
Issue to complete Zookeeper Quorum

### ISSUE THREAD

**Issue by jeqo** [jeqo/hashi-kafka#1]
```
[2019-11-14 21:06:45,730] WARN Interrupting SendWorker (org.apache.zookeeper.server.quorum.QuorumCnxManager)
[2019-11-14 21:06:45,730] INFO Notification: 1 (message format version), 2 (n.leader), 0x0 (n.zxid), 0x1 (n.round), LOOKING (n.state), 2 (n.sid), 0x0 (n.peerEpoch) LOOKING (my state) (org.apache.zookeeper.server.quorum.FastLeaderElection)
[2019-11-14 21:06:45,730] WARN Exception when using channel: for id 1 my id = 2 error = java.net.SocketException: Connection reset (org.apache.zookeeper.server.quorum.QuorumCnxManager)
[2019-11-14 21:06:45,731] WARN Send worker leaving thread (org.apache.zookeeper.server.quorum.QuorumCnxManager)
[2019-11-14 21:07:45,731] INFO Notification time out: 60000 (org.apache.zookeeper.server.quorum.FastLeaderElection)
[2019-11-14 21:07:45,732] INFO Notification: 1 (message format version), 2 (n.leader), 0x0 (n.zxid), 0x1 (n.round), LOOKING (n.state), 2 (n.sid), 0x0 (n.peerEpoch) LOOKING (my state) (org.apache.zookeeper.server.quorum.FastLeaderElection)
[2019-11-14 21:07:45,732] WARN Connection broken for id 1, my id = 2, error =  (org.apache.zookeeper.server.quorum.QuorumCnxManager)
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(SocketInputStream.java:210)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at org.apache.zookeeper.server.quorum.QuorumCnxManager$RecvWorker.run(QuorumCnxManager.java:1010)
[2019-11-14 21:07:45,732] WARN Interrupting SendWorker (org.apache.zookeeper.server.quorum.QuorumCnxManager)
[2019-11-14 21:07:45,732] WARN Exception when using channel: for id 1 my id = 2 error = java.net.SocketException: Broken pipe (Write failed) (org.apache.zookeeper.server.quorum.QuorumCnxManager)
[2019-11-14 21:07:45,732] WARN Send worker leaving thread (org.apache.zookeeper.server.quorum.QuorumCnxManager)
[2019-11-14 21:08:45,732] INFO Notification time out: 60000 (org.apache.zookeeper.server.quorum.FastLeaderElection)
[2019-11-14 21:08:45,733] WARN Connection broken for id 1, my id = 2, error =  (org.apache.zookeeper.server.quorum.QuorumCnxManager)
java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at org.apache.zookeeper.server.quorum.QuorumCnxManager$RecvWorker.run(QuorumCnxManager.java:1010)
[2019-11-14 21:08:45,733] WARN Interrupting SendWorker (org.apache.zookeeper.server.quorum.QuorumCnxManager)
[2019-11-14 21:08:45,733] WARN Exception when using channel: for id 1 my id = 2 error = java.net.SocketException: Connection reset (org.apache.zookeeper.server.quorum.QuorumCnxManager)
[2019-11-14 21:08:45,733] INFO Notification: 1 (message format version), 2 (n.leader), 0x0 (n.zxid), 0x1 (n.round), LOOKING (n.state), 2 (n.sid), 0x0 (n.peerEpoch) LOOKING (my state) (org.apache.zookeeper.server.quorum.FastLeaderElection)
[2019-11-14 21:08:45,734] WARN Send worker leaving thread (org.apache.zookeeper.server.quorum.QuorumCnxManager)
[2019-11-14 21:09:45,734] INFO Notification time out: 60000 (org.apache.zookeeper.server.quorum.FastLeaderElection)
[2019-11-14 21:09:45,735] INFO Notification: 1 (message format version), 2 (n.leader), 0x0 (n.zxid), 0x1 (n.round), LOOKING (n.state), 2 (n.sid), 0x0 (n.peerEpoch) LOOKING (my state) (org.apache.zookeeper.server.quorum.FastLeaderElection)
[2019-11-14 21:09:45,735] WARN Connection broken for id 1, my id = 2, error =  (org.apache.zookeeper.server.quorum.QuorumCnxManager)
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(SocketInputStream.java:210)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at org.apache.zookeeper.server.quorum.QuorumCnxManager$RecvWorker.run(QuorumCnxManager.java:1010)
```

**Comment by zhenik** [comment]
#2

**Comment by jeqo** [comment]
Fixed by #2
